<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Users</string>
  </property>
  <property name="windowIcon">
   <iconset resource="project51.qrc">
    <normaloff>:/images/circle.png</normaloff>:/images/circle.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QComboBox{
color:black;
border: none;
border-radius: 4px;
}
layout{
border:10px;
}
QRadioButton
{color:white;
}
QLabel
{color:white;
}
#Frame
{background:url(:/images/simple-background-blue-simple-minimalism-wallpaper-preview.jpg);
}
QPushButton
{background:white;
color:black;
border-radius:15px;
}
QPushButton:hover
{color:#333;
border-radius:15px;
background:#49ebff;
}
QTextEdit
{background:rgb(255, 255, 255);
border-radius:15px;
}
QLineEdit
{background:transparent;
border:none;
color: white;
border-bottom:1px solid #e2e2e2;
}</string>
  </property>
  <widget class="QTableWidget" name="table">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>281</width>
     <height>281</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <column>
    <property name="text">
     <string>Username</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Type</string>
    </property>
   </column>
  </widget>
  <widget class="QComboBox" name="cmb">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>191</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>100</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>380</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>110</y>
     <width>21</width>
     <height>281</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>110</y>
     <width>20</width>
     <height>281</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAdmin">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>280</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Change to Admin</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUser">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>280</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Change to User</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>130</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Change Type</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonAdmin">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>240</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Admin</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonUser">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>240</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>User</string>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>160</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Users Details üôç‚Äç</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDelete">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>330</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete this ID</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="project51.qrc"/>
 </resources>
 <connections/>
</ui>
